donut-app/resources-vivoactive_hr/settings/properties.xml

```diff
@@ -0,0 +1,13 @@
+<properties>
+    <property id="screenHeight" type="number">205</property>
```

Comment:
I would like to see the width and height properties swapped in order of appearance